dirty_cat/_super_vectorizer.py

```diff
@@ -104,19 +108,19 @@ class SuperVectorizer(ColumnTransformer):
         Note: currently, missing values are counted as a single unique value
```

Comment:
Should you add an `optional` for the `cardinality_threshold` as well?

Reply:
Yes, done!